spec/hyrax/transactions/steps/apply_collection_type_permissions_spec.rb

```diff
@@ -7,7 +7,8 @@
   let(:collection) do
     FactoryBot.valkyrie_create(:hyrax_collection,
                                title: "My Resource",
-                               collection_type_gid: collection_type_gid)
+                               collection_type_gid: collection_type_gid,
+                               with_permission_template: false)
```

Comment:
This is testing the ability of this step to add the permission template and associated initial state permissions.  It will fail if the permission template already exists.